openrtb_ext/imp_between.go

```diff
@@ -1,5 +1,8 @@
 package openrtb_ext
 
 type ExtImpBetween struct {
-	Host string `json:"host"`
+	Host        string `json:"host"`
+	PublisherID string `json:"publisher_id"`
```

Comment:
Please add `publisher_id` to `static/bidder-params/between.json` as a required parameter